### ISSUE TITLE
The model's performance seems terrible on my custom dataset

### ISSUE THREAD

**Issue by jackfromeast** [RonyAbecidan/ManTraNet-pytorch#9]
Hi!

I really appreciate your solid and excellent work! Recently, I just try to use the ManTraNet model to detect the tampering of my own dataset. My solution is to freeze the weights of IMTFE and try to fine-tune the weights of the anomaly detection network. But the performance on both train and test datasets seems quite terrible, that the f1 scores are less than 0.1 and loss didn't go down properly either. Do you have any suggestions? May I replace the detection network with other image segmentation networks?

**Comment by RonyAbecidan** [comment]
This does not surprise me. It really seems that this architecture is not very easy to train ! The authors of the paper may have concealed something about this training (we can at least notice that they didn't share the code enabling to train MantraNet on purpose). 

That being said, you can try to play with some hyperparameters like the batch size and the learning rate. If it does not work, your idea may also work ! You can't know without trying ;) 

If you succeed in getting a working way to train MantraNet please share with us what you've done if it's possible.
